langchain_mcp_adapters/client.py

```diff
@@ -271,4 +275,5 @@ async def __aexit__(
         exc_val: BaseException | None,
         exc_tb: TracebackType | None,
     ) -> None:
+        _, _, _ = exc_type, exc_val, exc_tb
```

Comment:
I don't think this statement is necessary?

Reply:
OK got it -- the way to do this is using a noqa comment and can add an extra inline comment to explain it's not being used.

it's better to avoid affecting run-time behavior to satisfy a linter

Reply:
No problem, I can make that change. 

I actually thought that this didn't affect runtime, but I guess python doesn't actually treat the `_` differently to a regular variable so a variable assignment is being made there.